### PR TITLE
467 use progressbar component from js-tools

### LINKED ISSUE #467: Use ProgressBar Component from onaio package
Currently we are [using a local ProgressBar component in the IRS Map page](https://github.com/onaio/reveal-frontend/blob/irs-reporting-table/src/containers/pages/IRS/Map/index.tsx#L25), which should be replaced by the `ProgressIndicators` package being introduced by onaio/js-tools#95.

- [ ] Install package ([@onaio/progress-indicators](https://github.com/onaio/js-tools/tree/master/packages/ProgressIndicators))
- [ ] Update component props to pass in correctly structured thresholds model

### PR DESCRIPTION
**Pull request by RayceeM**
Fixes #467

### CODE REVIEW

**Comment by moshthepitt** [comment]
@cKellyDesign - @RayceeM and I looked at it last week, I think she now has a way forward.  At least I hope so @RayceeM ?

File: src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -151,51 +151,34 @@ Array [
...
-          "color": "#FF4136",
-          "name": "Red",
-          "orEquals": true,
-          "value": 0.75,
-        },
-        "YELLOW_THRESHOLD": Object {
-          "color": "#FFDC00",
-          "name": "Yellow",
-          "value": 0.9,
-        },
```
Comment (moshthepitt) [inline src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap]: Do you think there is value in maintaining these older style thresholds?  For example, how would we now deal with a threshold that is `>= 80%`?

File: src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap [hunk 2]
```diff
@@ -151,51 +151,34 @@ Array [
...
     value={50}
   >
     <div
       className="progress"
-      style={
-        Object {
-          "height": "10px",
-          "marginBottom": "15px",
-        }
-      }
```
Comment (moshthepitt) [inline src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap]: Does the removal of this imply that we no longer need this CSS?

File: src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap (hunk 1 above)
Comment (RayceeM) [reply src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap]: if it's greater than the color value then it takes the default lineColor which is "#0000FF"

Comment (moshthepitt) [reply src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap]: @RayceeM I mean how will we handle the possibility of having `>=` as well as just `>` for the thresholds?

File: src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap (hunk 2 above)
Comment (RayceeM) [reply src/containers/pages/IRS/Map/tests/__snapshots__/index.test.tsx.snap]: We do, height is also a prop in the js library  that had a default value of `10px`

**Comment by cKellyDesign** [comment]
@RayceeM have you tried discarding the incoming changes to yarn.lock (from `master`) and re-running `yarn install` to resolve this conflict?

**Comment by RayceeM** [comment]
@cKellyDesign  no not yet, have been trying to start reveal but I keep getting an issue with `babel-loader` version

**Comment by cKellyDesign** [comment]
@RayceeM oh interesting, I was having issues with babel-loader yesterday in another project as well...

At any rate, you may try dumping your `node_modules` and `yarn.lock` file before re-installing to see if that clears things up.

```sh
rm -rf node_modules && rm -f yarn.lock && yarn install
```

**Comment by moshthepitt** [comment]
@RayceeM please update this branch.  A huge amount of stuff has changed :)

**Comment by RayceeM** [comment]
Here is a screenshot of a page using the progressbar component from js-library
nchelenge2

**Comment by RayceeM** [comment]
Screenshot of progressbar in a focus investigstion map using the js-component after passing lineColor
FI3

File: src/containers/pages/FocusInvestigation/map/active/index.tsx [hunk 3]
```diff
@@ -343,6 +349,16 @@ const SingleActiveFIMap = (props: MapSingleFIProps & RouteComponentProps<RoutePa
 
   const mapBounds = getMapBounds(jurisdiction);
 
+  function getLineColor(decimalValue: number) {
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/index.tsx:352]: Do you think that we should make [something like this](https://github.com/onaio/reveal-frontend/blob/8fcee000dc30c07ece47f898b9962b226cc92b0a/src/configs/settings.ts#L1617) for FI and then use it here?

Comment (RayceeM) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx:352]: Yeap, we can and pass `lineColorThreshold` instead of `lineColor`

Comment (moshthepitt) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx:352]: Cool, lets make one for Focus Investigations.  You can make it in the same settings file and can copy how we did it for [Namibia IRS](https://github.com/onaio/reveal-frontend/blob/8fcee000dc30c07ece47f898b9962b226cc92b0a/src/configs/settings.ts#L1644) if it makes sense.

File: src/containers/pages/FocusInvestigation/map/active/index.tsx [hunk 4]
```diff
@@ -432,7 +432,11 @@ const SingleActiveFIMap = (props: MapSingleFIProps & RouteComponentProps<RoutePa
                         ({goalReport.prettyPercentAchieved})
                       </p>
                       <br />
-                      <ProgressBar value={goalReport.percentAchieved} max={1} />
+                      <ProgressBar
+                        value={0.012}
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/index.tsx:436]: Looks like this is left over from testing?  We should revert back to `value={goalReport.percentAchieved}`

**Review by moshthepitt — CHANGES_REQUESTED** [review]

File: src/containers/pages/FocusInvestigation/map/active/index.tsx (hunk 4 above)
Comment (RayceeM) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx]: Yeah I have reverted back in the next commit

**Review by moshthepitt — APPROVED** [review]

**Review by p-netm — APPROVED** [review]

**Pull request merged**